tweets.py

```diff
@@ -196,6 +205,9 @@ def sort_and_print(urls):
     annotated_urls = extract_urls(user_tweets)
     pretty_print_urls = sort_and_print(annotated_urls)
     print(pretty_print_urls)
+    if USE_MAIL:
+        send_email(pretty_print_urls)
```

Comment:
you obviously didn't test this. this is not ok. You should always be 100% sure that your code works before you have someone else check it for you. Send email takes 2 parameters and here you only pass 1. 

to reiterate...NEVER SEND A PULL REQUEST FOR UNTESTED CODE!